docker/utility/wrap_chrome_binary.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+WRAPPER_PATH=$(readlink -f /usr/bin/google-chrome)
+BASE_PATH="$WRAPPER_PATH-base"
+mv "$WRAPPER_PATH" "$BASE_PATH"
+
+cat > "$WRAPPER_PATH" <<_EOF
+#!/bin/bash
+# Note: exec -a below is a bashism.
+exec -a "\$0" "$BASE_PATH" --no-sandbox "\$@"
+_EOF
+chmod +x "$WRAPPER_PATH"
```

Comment:
What is this wrapper for?

Reply:
allows for headless running of google-chrome for selenium and robot testing, getting an error without it being run in Dockerfile

Reply:
Hmm, I think we should look for a way to let chrome run with the sandbox feature turned on. For one thing, that will make it more similar to the other environments where we run robot tests. For another, turning off the sandbox sounds non-ideal since it's turning off a security feature, and a bit of googling suggests it is discouraged.

What was the error you were getting?

Reply:
https://ndportmann.com/chrome-in-docker/ discusses an approach that sounds better

Reply:
IIRC, this solves the problem that we needed (or thought we needed) the `--no-sandbox` option and we didn't want to modify our open_test_browser keyword in order to add it.

What I don't know is if we truly require that option. Some internet advice said we did, but I think that was advice from several years ago. I _think_ (but don't know for certain) that it's only required if you're running as root, which I think the docker container is doing. 

We probably should be creating a user inside the docker container instead of running as root. That might make the need for the option to go away, and with that we would no longer need the wrapper.

Reply:
<img width="1438" alt="Screen Shot 2019-12-30 at 5 05 14 PM" src="https://user-images.githubusercontent.com/28512644/71606543-9e21fa80-2b26-11ea-9c63-e5c0960bcaa7.png">

Reply:
Having dug into this for a while, I think adding `--no-sandbox` like this is the most pragmatic option. The container environment already provides one level of sandboxing. Sandboxing chrome from the rest of the container would be more critical if we were running tests from an untrusted source. We'll need to revisit this if we want to provide more isolation between MetaCI itself and the builds that it is running.